sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler_opencl.asciidoc

```diff
@@ -243,52 +385,45 @@ int main() {
 }
 ```
 
+=== Querying supported features and extensions
 
-== Issues
-
-* How should we expose the difference between OpenCL C versions?
-  It seems like there are two aspects to the problem.
-  Applications need some way to query which versions the backend (or device)
-  supports.
-  Applications also need some way to tell the runtime which version the kernel
-  is written in.
-+
---
-One option is to define separate enumerators in `source_language` for each
-version like this:
+This example demonstrates how to query the version of OpenCL C that is
+supported, how to query the supported features, and how to query the
+supported extensions.
 
 ```
-enum class source_language : /*unspecified*/ {
-  opencl_1_0,
-  opencl_1_1,
-  opencl_2_0,
-  opencl_3_0,
-};
+#include <iostream>
+#include <sycl/sycl.hpp>
+namespace syclex = sycl::ext::oneapi::experimental;
+
+int main() {
+  sycl::queue q;
+  sycl::device d =  q.get_device();
+
+  if (d.ext_oneapi_can_compile(syclex::source_language::opencl)) 
```

Comment:
```suggestion
  if (d.ext_oneapi_can_compile(syclex::source_language::opencl))
```

Kind of a nit, but there's a trailing space on this line and on several of the subsequent ones also.

Reply:
thanks. Got them.